build/tizen/visual-tests/CMakeLists.txt

```diff
@@ -17,10 +17,12 @@ FOREACH(VISUAL_TEST ${SUBDIRS})
   FILE(GLOB SRCS "${VISUAL_TESTS_SRC_DIR}/${VISUAL_TEST}/*.cpp")
   SET(SRCS ${SRCS} "${ROOT_SRC_DIR}/common/visual-test.cpp")
   ADD_EXECUTABLE(${VISUAL_TEST}.test ${SRCS})
-  TARGET_LINK_LIBRARIES(${VISUAL_TEST}.test ${REQUIRED_PKGS_LDFLAGS} -pie)
+  TARGET_LINK_LIBRARIES(${VISUAL_TEST}.test ${REQUIRED_PKGS_LDFLAGS} -ldl -pie)
```

Comment:
Why do we need ldl now? I thought you weren't using LD_LIBRARY_PATH anymore or is this for something else?

Reply:
done